### ISSUE TITLE
Add earthquake to earth and rocky planets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#72]
```
With a probability of 1 in two months. Damages buildings by 25% and power 
planets by 55%.
```

Original issue reported on code.google.com by `akarn...@gmail.com` on 23 Aug 2011 at 9:31

**Comment by GoogleCodeExporter** [comment]
```
Implemented in v0.92, updated in v0.93.200
```

Original comment by `akarn...@gmail.com` on 23 Aug 2011 at 9:31
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 23 Aug 2011 at 9:31
- Changed state: **Fixed**
